sagemaker-python-sdk/scikit_learn_model_registry_batch_transform/scikit_learn_model_registry_batch_transform.ipynb

```diff
@@ -0,0 +1,671 @@
+{
```

Comment:
<p>Style nit - use code formatting instead of plain text for <code>list_model_packages</code> ﻿</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3086/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
Fixed.